.github/workflows/js_api.yml

```diff
@@ -16,8 +16,10 @@ jobs:
                 include:
                     - conf: firefox
                       headless: --headless
+                      driver: --geckodriver ./geckodriver
                     - conf: chrome
                       headless: --headless
+            fail-fast: false
```

Comment:
it would be better to remove this line, in this way GitHub cancels all in-progress jobs if any matrix job fails.

Reply:
I know. But because of the firefox error I noticed it is actually useful to run all configurations to know if the problem is with only one or all of them.

Reply:
Aren't the configurations run in parallel? Can't we look at the GUI to detect which configuration has failed?

Reply:
They are run, but because of scheduling sometimes a configuration didn't start yet, so it will get canceled if another one fails.

Reply:
E.g. Firefox fails --> cancels chrome even, if it would have been fine